ReactiveCocoa/ControlSubject.swift

```diff
@@ -0,0 +1,44 @@
+import ReactiveSwift
+import enum Result.NoError
+
+/// Represents a subject of an interactive control.
+///
+/// Generally speaking, `ControlSubject` emits changes only for the user
+/// interactions in which the subject specializes.
+public final class ControlSubject<Value> {
```

Comment:
After reviewing my words, item (2) in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3210#discussion_r81995270 was actually false. Sorry about that.

-

Anyway, I think we might be able to reduce the problem into whether the one-value-for-one-event relationship is significant enough in practical.